apps/book/models.py

```diff
@@ -62,6 +62,24 @@ class LanguageType(models.TextChoices):
         NEPALI = 'nepali', 'Nepali'
         ENGLISH = 'english', 'English'
 
+    class Grade(models.TextChoices):
+        GRADE_ONE = 'grade_one', 'Grade one'
+        GRADE_TWO = 'grade_two', 'Grade two'
+        GRADE_THREE = 'grade_three', 'Grade three'
+        GRADE_FOUR = 'grade_four', 'Grade four'
+        GRADE_FIVE = 'grade_five', 'Grade five'
+        GRADE_SIX = 'grade_six', 'Grade six'
+        GRADE_SEVEN = 'grade_seven', 'Grade seven'
+        GRADE_EIGHT = 'grade_eight', 'Grade eight'
+        GRADE_NINE = 'grade_nine', 'Grade nine'
+        GRADE_TEN = 'grade_ten', 'Grade ten'
+        GRADE_ELEVEN = 'grade_eleven', 'Grade eleven'
+        GRADE_TWELVE = 'grade_twelve', 'Grade twelve'
+
+    class PublishType(models.TextChoices):
+        PUBLISH = 'publish', 'Publish'
```

Comment:
If it's just 2 states, then maybe use is_draft or is_published? cc: @frozenhelium @tnagorra

Reply:
After talking with @amarshahi, looks like just **is_draft** should be okay for now.

Reply:
@thenav56  Change to boolean field is_published